src/main/java/org/example/servletstexts/dao/TextsDAO.java

```diff
@@ -0,0 +1,35 @@
+package org.example.servletstexts.dao;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TextsDAO {
+
+    private int currentId = 0;
+    private final Map<Integer, String> texts = new HashMap<>();
+
+
+    public String get(int id) {
+        return texts.get(id);
+    }
+
+    public Map<Integer, String> getAll() {
+        return texts;
+    }
+
+    public int add(String text) {
+        currentId++;
+        texts.put(currentId, text);
+        return currentId;
+    }
+
+    public void delete(int id) {
+        texts.remove(id);
+    }
+
+    public void deleteAll() {
+        texts.clear();
+        currentId = 1;
```

Comment:
think about this case.
1 - you created text with id = 1
2 - you got this text id and send the id to someone who can get text by id. 
3 - you removed all texts.
4 - you created new text which you are not supposed  to see by someone.
5 - someone is trying to get text by id which you send to him (1)